assets/javascripts/discourse/templates/admin/plugins-explorer-index.hbs

```diff
@@ -0,0 +1,168 @@
+{{#if this.disallow}}
+  <h1>{{i18n "explorer.admins_only"}}</h1>
+{{else}}
+  {{#unless this.validQueryPresent}}
```

Comment:
I don't think you need this check anymore, it's been removed from the JS and it was only relevant when selecting a query on the main page